README.md

```diff
@@ -55,17 +55,17 @@ ssh, scp, rsync:
 ## Environment
 
 - Ubuntu 14.04
-- Drupal 7.43
-- MySQL 5.5.47
+- Drupal 7.50
+- MySQL 5.5.49
```

Comment:
@ruebot I will bring up an islandora_vagrant vs islandora_vagrant_base_box to verify version numbers I had one up and not the other.